packages/rocketchat-api/server/v1/groups.js

```diff
@@ -430,7 +435,16 @@ RocketChat.API.v1.addRoute('groups.setType', { authRequired: true }, {
 
 RocketChat.API.v1.addRoute('groups.unarchive', { authRequired: true }, {
 	post: function() {
+<<<<<<< HEAD
+		const findResult = findPrivateGroupByIdOrName({ roomId: this.bodyParams.roomId, userId: this.userId });
+
+		//The find method returns either with the group or the failure
+		if (findResult.statusCode) {
+			return findResult;
+		}
+=======
 		const findResult = findPrivateGroupById({ roomId: this.bodyParams.roomId, userId: this.userId, checkedArchived: false });
+>>>>>>> 0d6d3f546aa18bc0778b367186c13fb45ba1d6c2
```

Comment:
Please fix this merge conflict and test before you commit, that way you know it works and builds.